cmd/csi-node-driver-registrar/main.go

```diff
@@ -69,6 +70,7 @@ var (
 	httpEndpoint            = flag.String("http-endpoint", "", "The TCP network address where the HTTP server for diagnostics, including the health check indicating whether the registration socket exists, will listen (example: `:8080`). The default is empty string, which means the server is disabled. Only one of `--health-port` and `--http-endpoint` can be set.")
```

Comment:
Add pprof in flag description

Reply:
To make the sentence flow better, maybe something like "...diagnostics, including *pprof and* the health check..."

Reply:
Done!